porcupine/plugins/run/dialog.py

```diff
@@ -106,7 +106,15 @@ def __init__(self, file_path: Path, project_path: Path, suggestions: list[histor
         )
 
         sub_text = "\n".join("{%s} = %s" % pair for pair in substitutions.items())
-        ttk.Label(content_frame, text=f"Substitutions:\n{sub_text}\n").pack(fill="x")
+        sub_textbox = textutils.create_passive_text_widget(
+            content_frame, height=len(substitutions) + 2
```

Comment:
Looks like you deleted `width=1` by accident?

Reply:
I think I messed up something with git